mkosi/__init__.py

```diff
@@ -179,7 +179,7 @@ def configure_autologin(state: MkosiState) -> None:
             dropin.parent.mkdir(parents=True, exist_ok=True)
         with umask(~0o644):
             dropin.write_text(
-                """\
+                r"""\
                 [Service]
                 ExecStart=
                 ExecStart=-/sbin/agetty -o '-f -p -- \\u' --autologin root --noclear --keep-baud console 115200,38400,9600 $TERM
```

Comment:
Making the change to a raw string, will write an additional slash, i.e. the diff will be
```diff
--- old	2023-09-21 17:32:43.329891671 +0200
+++ new	2023-09-21 17:32:45.843252993 +0200
@@ -1,5 +1,6 @@
+\
 [Service]
 ExecStart=
-ExecStart=-/sbin/agetty -o '-f -p -- \u' --autologin root --noclear --keep-baud console 115200,38400,9600 $TERM
+ExecStart=-/sbin/agetty -o '-f -p -- \\u' --autologin root --noclear --keep-baud console 115200,38400,9600 $TERM
 StandardInput=tty
 StandardOutput=tty
```

Is that intended?

Reply:
Also note the additional slash at the beginning

Reply:
Right but we need the double slashes, the previous commit that moved these into the file actually introduced a bug. The first slash is a bug will fix